src/Uno.UWP/ApplicationModel/Resources/ResourceLoader.cs

```diff
@@ -48,6 +48,23 @@ public ResourceLoader()
 
 		public string GetString(string resource)
 		{
+			
+			// "/[file]/[name]" format support
+			if(resource.ElementAt(0) == '/')
+			{
+				int iInd;
```

Comment:
The naming convention used by Uno does not require the use of variable prefixes. In this case, this would do:
```suggestion
				int separatorIndex;
```

Reply:
"does not require" or "require to not"?

Reply:
Requires not to, bad wording sorry :)